docs/src/main/tut/patterns.md

```diff
@@ -6,13 +6,13 @@ permalink: /docs/rpc/patterns
 
 # Patterns
 
-So far so good, not too much code, no business logic, just a protocol definition with Scala annotations and generation IDL files from the scala definitions. Conversely, in this section, we are going to see how to complete our quickstart example. We'll take a look at both sides, the server and the client.
+So far so good, not too much code, no business logic, in the previous sections we have seen just some protocol definitions with Scala annotations and generation IDL files from the scala definitions. Conversely, in this section, we are going to see how to complete our quickstart example. We'll take a look at both sides, the server and the client.
```

Comment:
`we have seen just some protocol definitions with Scala annotations and generation IDL files from the scala definitions.` -> `we have seen some protocol definitions with Scala annotations and the generation of IDL files from the Scala definitions.`